connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -575,6 +583,8 @@ private void expectAdd(SourceSink sourceSink) throws Exception {
         worker.startTask(new ConnectorTaskId(CONNECTOR_NAME, 0), connectorConfig(sourceSink), generatedTaskProps, herder, TargetState.STARTED);
         EasyMock.expectLastCall().andReturn(true);
 
+        EasyMock.expect(herder.connectorType(CONNECTOR_NAME))
+            .andReturn(ConnectorType.SOURCE).anyTimes();
```

Comment:
Looks like `connectorType(String)` is again being used incorrectly with the _connector name_ rather than the _connector class name_.